go.mod

```diff
@@ -33,6 +33,7 @@ require (
 	github.com/satori/go.uuid v1.2.0 // indirect
 	github.com/shirou/gopsutil v2.19.9+incompatible // indirect
 	github.com/spf13/cobra v0.0.5 // indirect
+	github.com/syncthing/syncthing v0.10.26 // indirect
```

Comment:
[Nexus Lifecycle](https://www.sonatype.com/product-nexus-lifecycle) has found that this version of `syncthing` violates your company's policies.

Lifecycle recommends using version [v0.14.34](https://github.com/syncthing/syncthing/releases/tag/v0.14.34) instead as it does not violate any policies.